web/html/src/manager/systems/all-list.tsx

```diff
@@ -39,13 +40,14 @@ export function AllSystems(props: Props) {
     const removed = selectedSystems.filter((item) => !items.includes(item)).map((item) => [item, false]);
     const added = items.filter((item) => !selectedSystems.includes(item)).map((item) => [item, true]);
     const data = Object.assign({}, Object.fromEntries(added), Object.fromEntries(removed));
-    Network.post("/rhn/manager/api/sets/system_list", data);
+    Network.post("/rhn/manager/api/sets/system_list", data).catch(Network.showResponseErrorToastr);
 
     setSelectedSystems(items);
   };
 
   return (
     <>
+      <MessagesContainer />
```

Comment:
If it works, I would recommend moving this and the other container all the way up to whatever file is the `.renderer.tsx` for these views.